### ISSUE TITLE
Seemingly incorrect parsing of mill version

### ISSUE THREAD

**Issue by ckipp01** [scala-steward-org/scala-steward#3113]
I noticed today in the [Scala Center Scala Steward](https://github.com/scalacenter/steward/actions/runs/5470172748/jobs/9959938109#step:5:215) that Steward was failing on the [bloop-config](https://github.com/scalacenter/bloop-config) repo. When looking at the logs I see:

```
  org.scalasteward.core.io.process$ProcessFailedException: '"SBT_OPTS=-Xmx2048m -Xss8m -XX:MaxMetaspaceSize=512m" mill -i -p /home/runner/scala-steward/workspace/repos/scalacenter/bloop-config/scala-steward.sc show org.scalasteward.mill.plugin.StewardPlugin/extractDeps' exited with code 1.
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
  
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  
    0 57.5M    0  575k    0     0  1088k      0  0:00:54 --:--:--  0:00:54 1088k
  100 57.5M  100 57.5M    0     0  55.3M      0  0:00:01  0:00:01 --:--:--  111M
  Preparing Java 11.0.19 runtime; this may take a minute or two ...
  [info] compiling 1 Scala source to /home/runner/scala-steward/workspace/repos/scalacenter/bloop-config/out/mill-build/compile.dest/classes ...
  [info] done compiling
  Cannot resolve -p. Try `mill resolve _` to see what's available.
```
It fails with `-p` but it shouldn't be trying with a predef file since it's using `0.11.1` for its mill Version. It should be using `--import`. So somehow it's failing into here:

https://github.com/scala-steward-org/scala-steward/blob/b83aae55d9dd000548c3b3c9b63d79636e7b3c8b/modules/core/src/main/scala/org/scalasteward/core/buildtool/mill/MillAlg.scala#L48-L55

When it should be using:

https://github.com/scala-steward-org/scala-steward/blob/b83aae55d9dd000548c3b3c9b63d79636e7b3c8b/modules/core/src/main/scala/org/scalasteward/core/buildtool/mill/MillAlg.scala#L44-L47

**Comment by fthomas** [comment]
My guess is that it is fixed by #3095 which has not yet been released.

**Comment by fthomas** [comment]
I've just released 0.25.0 which includes the above fix.

**Comment by ckipp01** [comment]
> My guess is that it is fixed by https://github.com/scala-steward-org/scala-steward/pull/3095 which has not yet been released.

Ahh, sorry I didn't search well enough to realize that change wasn't released.

> I've just released 0.25.0 which includes the above fix.

Yup, this seems to have fixed it! Thanks!
